File: app/src/main/java/com/genius/wasylews/readme/di/module/FragmentModule.java
package com.genius.wasylews.readme.di.module;

import com.genius.wasylews.readme.presentation.main.fragment.BookPageFragment;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public interface FragmentModule {

    @ContributesAndroidInjector
    BookPageFragment bookPageFragment();
}

File: app/src/main/java/com/genius/wasylews/readme/di/module/FragmentModule.kt
package com.genius.wasylews.readme.di.module

import com.genius.wasylews.readme.presentation.book.BookFragment
import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
interface FragmentModule {

    @ContributesAndroidInjector
    fun bookFragment(): BookFragment
}